src/assets/styles/main.scss

```diff
@@ -257,18 +228,36 @@ $icon-font-path: '../icon-font' !default;
     }
   }
 
+  .vjs-control {
+    width: 2.5em;
+  }
+
+  .vjs-control::before,
+  .vjs-icon-placeholder:before {
+    font-size: 1.8em;
+    line-height: 1.7;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    text-rendering: optimizeLegibility;
+  }
+
   .vjs-control-bar {
     z-index: 2;
+    backdrop-filter: blur(10px);
+    font-size: 120%;
+    // responsive controls sizes, based on container width
+    // font-size: clamp(100%, 2cqw, 150%);
```

Comment:
```suggestion
```

Reply:
I created responsive sized icons mechanism, but we decided not to push it.
I left it in place for now in case we change our mind, wanted to see what others have to say.